frameworks/cassandra/universe/resource.json

```diff
@@ -5,7 +5,8 @@
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
       "scheduler-zip": "{{artifact-dir}}/cassandra-scheduler.zip",
       "bootstrap-zip": "{{artifact-dir}}/bootstrap.zip",
-      "executor-zip": "{{artifact-dir}}/executor.zip"
+      "executor-zip": "{{artifact-dir}}/executor.zip",
+      "awscli-zip": "https://s3cli-artifacts.s3.amazonaws.com/s3cli-0.0.55-linux-amd64"
```

Comment:
who owns this bucket? should it be copied into downloads.mesosphere.com? (which is confusingly backed by a bucket named downloads.mesosphere.io)